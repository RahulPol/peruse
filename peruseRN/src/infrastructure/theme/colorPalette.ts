import { Color } from '../typings/colors';

export const lightColorPalette: Color = {
  brand: {
    primary: '#DB4818',
    muted: '#565656',
  },
  ui: {
    primary: '#FF6721',
    secondary: '#CCCCCC',
    tertiary: '#565656',
    disabled: '#999999',
    error: '#D0421B',
    success: '#138000',
  },
  bg: {
    primary: '#FFFFFF',
    secondary: '#F1F1F1',
  },
  text: {
    primary: '#FFFFFF',
    secondary: '#9E9E9E',
    disabled: '#999999',
    error: '#D0421B',
    success: '#138000',
  },
};

export const darkColorPalette: Color = {
  ...lightColorPalette,
};
