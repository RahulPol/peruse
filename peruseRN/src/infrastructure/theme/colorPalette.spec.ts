import { customColorPalette } from './colorPalette';

describe('ColorPalette', () => {
  it('should create a color palette with custom colors', () => {
    expect(customColorPalette).toBeDefined();
    expect(customColorPalette).toMatchObject({
      primary: '#FF6721',
      onPrimary: '#ffffff',
      secondary: '#9e9e9e',
      tertiary: '#565656',
    });
  });
});
