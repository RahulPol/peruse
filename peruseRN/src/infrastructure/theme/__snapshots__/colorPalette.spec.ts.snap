// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPalette should create a color palette with custom colors 1`] = `
Object {
  "bg": Object {
    "primary": "#FFFFFF",
    "secondary": "#F1F1F1",
  },
  "brand": Object {
    "muted": "#565656",
    "primary": "#DB4818",
  },
  "text": Object {
    "disabled": "#999999",
    "error": "#D0421B",
    "primary": "#FFFFFF",
    "secondary": "#9E9E9E",
    "success": "#138000",
  },
  "ui": Object {
    "disabled": "#999999",
    "error": "#D0421B",
    "primary": "#FF6721",
    "secondary": "#CCCCCC",
    "success": "#138000",
    "tertiary": "#565656",
  },
}
`;
